app/controllers/sessions_controller.rb

```diff
@@ -1,30 +1,67 @@
 class SessionsController < ApplicationController
+  skip_before_action :require_login, only: [:create]
+
```

Comment:
I often find it useful to not require the user to be logged in to log out. That way if I the developer make a mistake and get the user ID in my session out of sync with my database, I can reset it to `nil` even if the `require_login` method doesn't pass.